membership-attribute-service/app/services/PaymentFailureAlerter.scala

```diff
@@ -0,0 +1,85 @@
+package services
+
+import java.util.Locale
+
+import com.gu.memsub.subsv2.Subscription
+import com.gu.memsub.subsv2.SubscriptionPlan.AnyPlan
+import com.gu.zuora.api.{RegionalStripeGateways, StripeAUMembershipGateway, StripeUKMembershipGateway}
+import com.gu.zuora.rest.ZuoraRestService.{AccountObject, AccountSummary, PaymentMethodId, PaymentMethodResponse}
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+
+import scala.concurrent.{ExecutionContext, Future}
+import scalaz.syntax.std.boolean._
+import scalaz.{Disjunction, \/}
+
+
+object PaymentFailureAlerter {
+
+  private def accountObject(accountSummary: AccountSummary) =
+    AccountObject(
+      Id = accountSummary.id,
+      Balance = accountSummary.balance,
+      Currency = accountSummary.currency,
+      DefaultPaymentMethodId = accountSummary.defaultPaymentMethod.map(_.id),
+      PaymentGateway = accountSummary.billToContact.country.map(RegionalStripeGateways.getGatewayForCountry)
+    )
+
+
+  def membershipAlertText(accountSummary: AccountSummary, subscription: Subscription[AnyPlan],
```

Comment:
I'd do it as one parameter per line, indended two spaces.  Then renaming the method won't mess any alighnment, also adding/removing parameters will be fairly clear.